compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/OnCanvasTests.kt

```diff
@@ -34,30 +34,36 @@ import org.w3c.dom.events.Event
 
 private const val canvasId: String = "canvasApp"
 
+private external interface CanReplaceChildren {
+    // this is a standard method for (among other things) emptying DOM element content
+    // https://developer.mozilla.org/en-US/docs/Web/API/Element/replaceChildren
+    // TODO: add this to our kotlin web external definitions
+    fun replaceChildren()
+}
+
 internal interface OnCanvasTests {
 
     @BeforeTest
     fun beforeTest() {
+        /** TODO: [kotlin.test.AfterTest] is fixed only in kotlin 2.0
+        see https://youtrack.jetbrains.com/issue/KT-61888
+         */
         resetCanvas()
     }
 
-    fun getCanvas() = document.getElementById(canvasId) as HTMLCanvasElement
-
     private fun resetCanvas() {
-        /** TODO: [kotlin.test.AfterTest] is fixed only in kotlin 2.0
-        see https://youtrack.jetbrains.com/issue/KT-61888
-         */
-        document.getElementById(canvasId)?.remove()
+        (getCanvasContainer() as CanReplaceChildren).replaceChildren()
```

Comment:
`getCanvasContainer()` returns HTMLCanvasElement. And here we call `replaceChildren()` on it. Why is it needed? I thought that our canvas doesn't have any children.

Reply:
@eymar  getCanvasContainer returns the parent container (which is div)

Reply:
I see. I got confused by the name of the variable - `canvasId` used in `getElementById`.

Reply:
@eymar you confusion is reasonable, I pushed commit that changes the name of the canvasId to containerId